OrbitCore/LinuxUtils.cpp

```diff
@@ -86,39 +88,52 @@ std::string ExecuteCommand(const char* a_Cmd) {
 //-----------------------------------------------------------------------------
 void ListModules(pid_t a_PID,
                  std::map<uint64_t, std::shared_ptr<Module>>& o_ModuleMap) {
-  std::map<std::string, std::shared_ptr<Module>> modules;
+  std::map<std::string, std::pair<uint64_t, uint64_t>> module_addresses;
   std::vector<std::string> result = ListModules(a_PID);
+
   for (const std::string& line : result) {
     std::vector<std::string> tokens = Tokenize(line);
```

Comment:
absl::StrSplit? I wanted to do it for a while now.. but since you are here...

Reply:
done